From b6d435a7401e2b0a39d37321cf91eaf048d42ae3 Mon Sep 17 00:00:00 2001
From: Arthur Oliveira <arthur.oliveira@ossystems.com.br>
Date: Fri, 3 Nov 2023 10:10:37 -0300
Subject: [PATCH] python3-zopeevent: Add Zope.Event Python Recipe

This commit adds the python3-zopeevent recipe, which provides a simple event
system. It is licensed under ZPL-2.1.

Signed-off-by: Arthur Oliveira <arthur.oliveira@ossystems.com.br>
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 .../recipes-devtools/python/python3-zopeevent_5.0.bb   | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 meta-python/recipes-devtools/python/python3-zopeevent_5.0.bb

diff --git a/meta-python/recipes-devtools/python/python3-zopeevent_5.0.bb b/meta-python/recipes-devtools/python/python3-zopeevent_5.0.bb
new file mode 100644
index 000000000..4d914c63e
--- /dev/null
+++ b/meta-python/recipes-devtools/python/python3-zopeevent_5.0.bb
@@ -0,0 +1,10 @@
+DESCRIPTION = "The zope.event package provides a simple event system"
+
+LICENSE = "ZPL-2.1"
+LIC_FILES_CHKSUM = "file://PKG-INFO;beginline=8;endline=8;md5=72092419572155ddc2d4fb7631c63dd3"
+
+SRC_URI[sha256sum] = "bac440d8d9891b4068e2b5a2c5e2c9765a9df762944bda6955f96bb9b91e67cd"
+
+PYPI_PACKAGE = "zope.event"
+
+inherit pypi setuptools3
-- 
2.34.1

